### PR TITLE
HA: Add missing Icinga 2 status information to heartbeat/icingadb_instance

### PR DESCRIPTION
**Pull request by N-o-X**

### CODE REVIEW

File: ha/heartbeat.go [hunk 1]
```diff
@@ -79,7 +92,17 @@ func IcingaHeartbeatListener(rdb *connection.RDBWrapper, chEnv chan *Environment
 						Name:     app.Environment,
 						ID:       Sha1bytes([]byte(app.Environment)),
 						NodeName: app.NodeName,
-						Icinga2:  Icinga2Info{app.Version, app.ProgramStart, nil},
+						Icinga2: Icinga2Info{
+							app.Version,
+							app.ProgramStart,
+							utils.EncodeChecksum(app.EndpointId),
```
Comment (Al2Klimov) [inline ha/heartbeat.go]: What is this change for?

File: ha/ha.go [hunk 2]
```diff
@@ -69,29 +69,71 @@ func (h *HA) updateOwnInstance(env *Environment) error {
...
+		utils.Bool[env.Icinga2.EventHandlersEnabled],
+		utils.Bool[env.Icinga2.FlapDetectionEnabled],
+		utils.Bool[env.Icinga2.PerformanceDataEnabled],
+		h.super.EnvId,
+	)
 	return err
 }
 
 func (h *HA) insertInstance(env *Environment) error {
+	log.Info(env.Icinga2)
```
Comment (Al2Klimov) [inline ha/ha.go]: C'mon, srsly?

File: ha/ha.go [hunk 3]
```diff
@@ -69,29 +69,71 @@ func (h *HA) updateOwnInstance(env *Environment) error {
...
 	_, err := h.super.Dbw.SqlExec(
 		mysqlObservers.updateIcingadbInstanceByEnvironmentId,
 		"UPDATE icingadb_instance SET id = ?, endpoint_id = ?, heartbeat = ?,"+
-			" icinga2_version = ?, icinga2_start_time = ? WHERE environment_id = ?",
-		h.uid[:], env.Icinga2.EndpointId, h.lastHeartbeat, env.Icinga2.Version,
-		int64(env.Icinga2.ProgramStart*1000), h.super.EnvId)
+			" icinga2_version = ?, icinga2_start_time = ?, icinga2_notifications_enabled = ?,"+
+			" icinga2_active_service_checks_enabled = ?, icinga2_active_host_checks_enabled = ?,"+
+			" icinga2_event_handlers_enabled = ?, icinga2_flap_detection_enabled = ?,"+
+			" icinga2_performance_data_enabled = ? WHERE endpoint_id = ?",
```
Comment (Al2Klimov) [inline ha/ha.go]: `endpoint_id`?

File: ha/heartbeat.go (hunk 1 above)
Comment (N-o-X) [reply ha/heartbeat.go]: We previously didn't set the endpoint_id, because the Icinga 2 part of that was missing.

**Pull request merged**